.github/workflows/analyze.yml

```diff
@@ -55,7 +55,7 @@ jobs:
           name: bundle_analysis.json
 
       - name: Download base branch bundle stats
-        uses: dawidd6/action-download-artifact@v2
+        uses: dawidd6/action-download-artifact@v6
```

Comment:
do we really need an external action for this? could we rewrite this to use https://github.com/actions/toolkit/tree/main/packages/artifact instead?